docs/configuration_options.md

```diff
@@ -3604,3 +3604,147 @@ Store adaptive timeout metrics in Argus. Disabled for performance tests only.
 **default:** True
 
 **type:** boolean
+
+
+## **cloud_api_url** / SCT_CLOUD_API_URL
+
+Scylla Cloud API endpoint URL (e.g., https://app.ext.lab.scylla.cloud/api)
+
+**default:** N/A
+
+**type:** str (appendable)
+
+
+## **cloud_api_token** / SCT_CLOUD_API_TOKEN
+
+Scylla Cloud API authentication token
+
+**default:** N/A
+
+**type:** str (appendable)
+
+
+## **cloud_account_id** / SCT_CLOUD_ACCOUNT_ID
+
+Scylla Cloud account ID
+
+**default:** N/A
+
+**type:** int
+
+
+## **cloud_credentials_path** / SCT_CLOUD_CREDENTIALS_PATH
+
+Path to Scylla Cloud credentials file, if stored locally
+
+**default:** N/A
+
+**type:** str (appendable)
+
+
+## **cloud_env** / SCT_CLOUD_ENV
+
+Scylla Cloud environment (e.g., lab).<br>WARNING: environment specific credentials will be read from keystore (s3 or local file) and<br>will override 'cloud_api_url', 'cloud_api_token' and 'cloud_account_id' parameters
+
+**default:** N/A
+
+**type:** str (appendable)
+
+
+## **cloud_provider** / SCT_CLOUD_PROVIDER
+
+Cloud provider for Scylla Cloud deployment (aws, gcp)
+
+**default:** N/A
+
+**type:** str (appendable)
+
+
+## **cloud_region** / SCT_CLOUD_REGION
```

Comment:
it can reuse the existing backends configurations, no need to introduce new ones here.

same goes to instance types

Reply:
those are no in use, in the end it was align that it's `backend: aws` and `db_type: scylla-cloud`
so the rest of the machinery in SCT works the same.

I don't think it complicated things too much, it's just a handful of one that we'll need to translate 
I don't like the idea of having one configuration item, that can hold either instance types, or different regions type from different clouds
also for regions it would mean extra pipeline parameters to expose for the new cloud regions